src/test/java/BasePge.java

```diff
@@ -0,0 +1,43 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class BasePge {
```

Comment:
A package in Java is used to group related classes. 

For example package "pages" - Includes the Page Objects (or Page Classes)

For example

![image](https://github.com/Test-Pro-Bootcamp/A53-QA-Automation-Framework/assets/104761482/8270734d-e3af-463e-bbbf-063f9a21965e)